tests/test_utc_tai_tdb_tt.py

```
@@ -282,187 +276,196 @@ def runTest(self):
+                tdb = time_from_day_sec(daylist, 43200., 'TDB', leapsecs=False)
+                (day, sec) = day_sec_from_time(tdb, 'TDB', leapsecs=True)
+                tdb2 = time_from_day_sec(day, sec, 'TDB', leapsecs=True)
+                (dtest, stest) = day_sec_from_time(tdb2, 'TDB', leapsecs=False)
+                assert np.all(dtest == daylist)
+                assert np.abs(stest - 43200.).max() < 2.e-7
+
+    finally:
+        try:
+            cspyce.kclear()
+        except NameError:
+            pass
```

Ruff (0.15.6)

[warning] Use `contextlib.suppress(NameError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(NameError): ...`

(SIM105)